src/test/java/callhub/connect/controllers/FileControllerTests.java

```diff
@@ -120,4 +125,58 @@ void testFindDocumentByID_InvalidID() throws Exception {
         RequestBuilder request = get("/files/12345678");
         mockMvc.perform(request).andExpect(status().isBadRequest()).andExpect(content().string("File could not be found."));
     }
+
+    @Test
+    void testPDFLinkValidSession() throws Exception {
+        String currentDirectory = System.getProperty("user.dir");
+        String pathName = currentDirectory + "/src/main/java/callhub/connect/pdfs/fall2023.pdf";
+        File pdfFile = new File(pathName);
+        MockMultipartFile sampleFile = new MockMultipartFile(
+                "file",
+                pdfFile.getName(),
+                "application/pdf",
+                new FileInputStream(pdfFile));
+
+
+        when(documentRepository.save(any())).thenReturn(new FileDocument(
+                "MockFile",
+                new Binary("sample pdf content".getBytes()),
+                LocalDate.now()));
```

Comment:
Missing Import for LocalDate: 
- Similar to the other file, I would add an import statement for `java.time.LocalDate` if it's being used in the test cases.